h/static/scripts/guest.coffee

```diff
@@ -119,9 +120,27 @@ class Annotator.Guest extends Annotator
       # Announce the new positions, so that the sidebar knows
       this.plugins.Bridge.sync([highlight.annotation])
 
+  # Utility function to remove the hash part from a URL
+  _removeHash: (url) ->
+    url = new URL url
```

Comment:
This API is not the most portable: https://developer.mozilla.org/en/docs/Web/API/URL#Browser_compatibility
Can you conceive of an alternative or include a polyfill?